packages/main/src/Input.js

```diff
@@ -826,7 +826,6 @@ class Input extends UI5Element {
 		}
 
 		if (!itemPressed) {
-			this.fireEventByAction(this.ACTION_ENTER, event);
```

Comment:
@ilhan007 do you remember why we have fired enter manually? I did not find any place it is handled

Reply:
Yes, indeed it's not handled anymore in fireEventByAction, it used to be handled for sure, but I don't quite remember why - I think it was related to IE, so if we want to find out what wash this about I would start looking in the changes related to the IE clean up, but I think we can just remove this line